sql/2021/resource-hints/consoleLog_incorrect_type.sql

```diff
@@ -0,0 +1,34 @@
+#standardSQL
+# returns the number of pages which preload a resource of the incorrect script type
+
+# helper to create percent fields
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
```

Comment:
It's preferred to calculate the raw quotient in SQL and format it as a percent in Sheets. This gives you more control over the number of decimal places depending on the values. So I'd suggest removing it from all of these queries.

Also this function appears to be unused here.

Reply:
I updated already in https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide#formatting-results but just added a comment about rounding too as see I hadn't explicitly mentioned that.